lib/imap.php

```diff
@@ -67,8 +67,20 @@ public function checkPassword($uid, $password) {
  		}
 
 		$mbox = @imap_open($this->mailbox, $username, $password, OP_HALFOPEN, 1);
-		imap_errors();
-		imap_alerts();
+		$imapErrors = imap_errors();
+		$imapAlerts = imap_alerts();
+		if (!empty($imapErrors) {
```

Comment:
there is a closing ')' missing

Reply:
Whoops, just fixed this. Nice to know it works as intended 😄